### ISSUE TITLE
[Feature request] Possible to prioritize starred items?

### ISSUE THREAD

**Issue by amatiasq** [SebastianMC/obsidian-custom-sort#45]
`Starred` core plugin allows to "star" a note.

I'm looking for a plugin that allows me to sort by `> advanced created` but placing starred notes first (a folder considered starred if a note inside is starred).

Would that be possible with this plugin?

**Comment by SebastianMC** [comment]
Hi @amatiasq 

in result of your request I examined the construction of the core plugin 'starred'.
It seems that integration is possible and the 'starred' plugin can be queried '_is the file ... starred?_'
Next step is a simple PoC to confirm the above finding

**Comment by SebastianMC** [comment]
Hi @amatiasq 

the feature is available in the newest ([1.5.0](https://github.com/SebastianMC/obsidian-custom-sort/releases/tag/1.5.0)) release of the plugin.

There is a new keyword `starred:` and it can be used as in the below example to place the starred items in the top of File Explorer:

```yaml
---
sorting-spec: |
  target-folder: /*
  > advanced created 
  starred:
---
```

This is a concise versions of the configuration. For a more verbose version with explanatory in-line comments please refer to the [documentation of the plugin](https://github.com/SebastianMC/obsidian-custom-sort/blob/master/docs/manual.md#matching-starred-items).

**Comment by amatiasq** [comment]
That's impressively fast! I've updated the plugin and works as expected.

![Screenshot 2023-01-03 at 19 31 46](https://user-images.githubusercontent.com/1533589/210419369-a9ab8603-533d-41c8-92f3-840b9cd8d4cc.png)

Thank you @SebastianMC 🧙
